ConsoleApp/MyStack/MyStack.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp.MyStack
+{
+    internal class MyStack<T> : IEnumerable<T>
+    {
+        private Element<T> _head;
+        private int _count;
+
+        public int GetCount
+        {
+            get { return _count; }
+            set { }
+        }
+
+        public MyStack()
```

Comment:
initialization is not required here because by default head will be initialized with null & count with 0